test/k8sT/StressPolicy.go

```diff
@@ -0,0 +1,121 @@
+
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NightlyPolicyStress", func() {
+
+	var (
+		kubectl          *helpers.Kubectl
+		backgroundCancel context.CancelFunc = func() {}
+		backgroundError  error
+	)
+
+	BeforeAll(func() {
+		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		deploymentManager.SetKubectl(kubectl)
+	})
+
+	JustBeforeEach(func() {
+		backgroundCancel, backgroundError = kubectl.BackgroundReport("uptime")
+		Expect(backgroundError).To(BeNil(), "Cannot start background report process")
+	})
+
+	JustAfterEach(func() {
+		blacklist := helpers.GetBadLogMessages()
+		kubectl.ValidateListOfErrorsInLogs(CurrentGinkgoTestDescription().Duration, blacklist)
+		backgroundCancel()
+	})
+
+	AfterEach(func() {
+		deploymentManager.DeleteAll()
+	})
+
+	AfterFailed(func() {
+		kubectl.CiliumReport(helpers.CiliumNamespace,
+			"cilium endpoint list")
+	})
+
+	AfterAll(func() {
+		deploymentManager.DeleteCilium()
+		kubectl.CloseSSHClient()
+	})
+
+	newlineRegexp := regexp.MustCompile(`\n[ \t\n]*`)
+	trimNewlines := func(script string) string {
+		return newlineRegexp.ReplaceAllLiteralString(script, " ")
+	}
+	// Return a command string for bash loop.
+	loopCommand := func(cmd, namespace string, count int) string {
+		// Repeat 'cmd' 'count' times.
+		// '${namespace} in 'cmd' will be replaced by 'namespace'.
+		// '$i' in 'cmd' will be replaced by the loop counter, starting from 1.
+		//
+		// Note: All newlines and the following whitespace is removed from the script below.
+		//       This requires explicit semicolons also at the ends of lines!
+		return trimNewlines(fmt.Sprintf(
+			`/bin/bash -c
+			'namespace=%s;
+                        for i in $(seq 1 %d); do
+			  %s;
+			done'`,
+			namespace, count, cmd))
+	}
+
+	Context("Identity Churn", func() {
+		It("Run policy-stress-test manifest with identity churn", func() {
+			deploymentManager.Deploy(helpers.CiliumNamespace, StatelessEtcd)
+			deploymentManager.WaitUntilReady()
+
+			host, port, err := kubectl.GetServiceHostPort(helpers.CiliumNamespace, "stateless-etcd")
+			Expect(err).Should(BeNil(), "Unable to retrieve ClusterIP and port for stateless-etcd service")
+
+			etcdService := fmt.Sprintf("http://%s:%d", host, port)
+			opts := map[string]string{
+				"global.etcd.enabled":           "true",
+				"global.etcd.endpoints[0]":      etcdService,
+				"global.identityAllocationMode": "kvstore",
+				"global.prometheus.enabled":     "true",
+			}
+			if helpers.ExistNodeWithoutCilium() {
+				opts["config.synchronizeK8sNodes"] = "false"
+			}
+			deploymentManager.DeployCilium(opts, DeployCiliumOptionsAndDNS)
+
+			randomNamespace := deploymentManager.DeployRandomNamespace(ConnectivityCheck)
+			deploymentManager.WaitUntilReady()
+
+			// Insert 20000 identities, half matching a fromEndpoints in policy, half matching any wildcarded fromEndpoints
+			ciliumPod, err := kubectl.GetCiliumPodOnNodeWithLabel(helpers.CiliumNamespace, helpers.K8s1)
```

Comment:
Same here for `kubectl.GetCiliumPodOnNodeWithLabel`:

```suggestion
			ciliumPod, err := kubectl.GetCiliumPodOnNodeWithLabel(helpers.K8s1)
```

Reply:
Fixed, thanks!